plexapi/video.py

```diff
@@ -613,31 +622,34 @@ def seasonNumber(self):
         return self.index
 
     def episodes(self, **kwargs):
-        """ Returns a list of :class:`~plexapi.video.Episode` objects. """
+        """ Returns a list of :class:`~plexapi.video.Episode` objects in the season. """
         key = '/library/metadata/%s/children' % self.ratingKey
-        return self.fetchItems(key, **kwargs)
+        return self.fetchItems(key, Episode, **kwargs)
 
     def episode(self, title=None, episode=None):
         """ Returns the episode with the given title or number.
 
             Parameters:
                 title (str): Title of the episode to return.
-                episode (int): Episode number (default:None; required if title not specified).
+                episode (int): Episode number (default: None; required if title not specified).
+
+            Raises:
+                :exc:`~plexapi.exceptions.BadRequest`: If title or episode parameter is missing.
         """
-        if not title and not episode:
-            raise BadRequest('Missing argument, you need to use title or episode.')
         key = '/library/metadata/%s/children' % self.ratingKey
         if title:
-            return self.fetchItem(key, title=title)
-        return self.fetchItem(key, parentIndex=self.index, index=episode)
+            return self.fetchItem(key, Episode, title__iexact=title)
+        elif episode:
```

Comment:
Same here `episode=0` won't work. Probably `elif episode is not None` or `elif isinstance(episode, int)`.

https://i.imgur.com/FrPLgt8.png